src/tests/ftest/cart/test_group_np_srv.c

```diff
@@ -77,10 +114,13 @@ test_run(d_rank_t my_rank)
 		DBG_PRINT("Group config file saved\n");
 	}
 
-	if (test_g.t_hold)
+
+	if (test_g.t_hold) {
 		sleep(test_g.t_hold_time);
+	}
```

Comment:
no need for { } around single statement

Reply:
17af1bf3c715de43e313528cc240abd69a8346f2